src/ipfsd-in-proc.js

```diff
@@ -29,11 +30,11 @@ class Node {
     this.path = this.opts.repoPath
     this.repo = createRepo(this.path)
     this.disposable = this.opts.disposable
+    this.initialized = fs.existsSync(this.path)
```

Comment:
this might fail in the browser, since it wouldn't be a path... Maybe we can use the `ipfs-repo` modules here to verify it's a repo - tho I've little experience with it.

Reply:
Whew! Took me a minute to realize we use package.json to define aliases for some files on browsers. That threw me for a loop.

There's a good possibility from `ipfs-repo`: `_isInitialized(callback)`. I'm about to push something, just running all tests. Lemme know what you think.